### DISCUSSION TITLE
Grid Feature Buttons are Red?

### DISCUSSION THREAD

**Discussion by bludog** [IgniteUI/app-builder#24] [Q&A] [answered]
My Grid Feature buttons and Grid Rows are showing Red in App Builder? I have not seen this in any examples or tutorials? When I export the code it also shows red?
![Screenshot 2023-03-02 110658](https://user-images.githubusercontent.com/5618166/222502023-38169fed-cfb9-4d5f-8d77-48a50cf86554.png)

Is there any way to have it set to the default color (black)?

**Comment by zdrawku** [comment]
Hi @bludog, looking at the provided image, it looks like the color is with the expected hex value #e41b75. Could you please inspect the elements and share the hex?

**Comment by bludog** [comment]
**Here is the HTML from the "Advanced Filtering" button**
`<button` igxbutton="outlined" type="button" name="btnAdvancedFiltering" igxripple="" title="Open advanced filtering dialog" class="igx-grid-toolbar__adv-filter igx-button igx-button--outlined igx-button--cosy" role="button"><igx-icon class="material-icons igx-icon" aria-hidden="true">filter_list</igx-icon><span></span><span class="ng-star-inserted">Advanced filtering</span></button>`

Forgive me... I do not see the Hex value? I am not a great front end (css/javascript) developer that is why I am using "App Builder". I do not see this red in any examples? Are you saying this is the normal color?

**Reply by zdrawku** [reply, marked as answer]
Apologies for the late reply, yes, that's the expected color. The "redish" perception might be due to Monitor (screen) configurations causing the colors to be more saturated
